examples/CURL/CMakeLists.txt

```diff
@@ -5,15 +5,21 @@ cmake_minimum_required(VERSION 3.0)
 
 # Emulate HunterGate:
 # * https://github.com/hunter-packages/gate
+set(TESTING_CONFIG_OPT FILEPATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Hunter/config.cmake) # Force OpenSSL 1.1.0a
```

Comment:
This file is missing. Also since it's `FILEPATH` we can use any location. It can be simply `${CMAKE_CURRENT_SOURCE_DIR}/config.cmake`